guides/prow_knative_setup.md

```diff
@@ -0,0 +1,89 @@
+# Prow setup for Knative projects
```

Comment:
Is this just copy/pasted from original place?

Reply:
Mostly, but also with a couple of changes and fixes.